CHANGELOG.md

```diff
@@ -16,9 +18,16 @@
 
 ## Enhancements
 
+- Updated chai and eslint dependencies
```

Comment:
This is not needed. Changelog is strictly for Users of the library. Not developers.